docs/source/resources/scylladbclusters/basics.md

```diff
@@ -0,0 +1,218 @@
+# ScyllaDBClusters
+
+## Introduction
+
+The [ScyllaDBCluster](../../api-reference/groups/scylla.scylladb.com/scylladbclusters.rst) resource defines a multi-datacenter ScyllaDB cluster that can span multiple geo-distributed Kubernetes clusters.
+This section provides an overview of its structure and demonstrates how to perform basic configurations and access the APIs.
+It is not intended as a comprehensive guide to all capabilities. For a full list of available options, refer to the [generated API reference](../../api-reference/groups/scylla.scylladb.com/scylladbclusters.rst).
+
+
+::::{tip}
+You can always see the currently supported API fields for a particular version installed in your cluster by running
+:::{code}
+kubectl explain --api-version='scylla.scylladb.com/v1alpha1' ScyllaDBCluster.spec
+:::
+::::
+
+## Creating a ScyllaDBCluster
+
+### Prerequisites
+
+This document outlines the process of deploying a Multi-Datacenter ScyllaDB cluster. Before proceeding, ensure that the necessary infrastructure is in place.
+We assume you have four interconnected Kubernetes clusters that can communicate with each other using Pod IPs. These clusters are categorized as follows:
```

Comment:
nit: PodIPs ?

Reply:
I didn't mean to point out particular expose option, so leaving as is.